<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSLR_Shooter_Window</class>
 <widget class="QMainWindow" name="DSLR_Shooter_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSLR Shooter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QToolButton" name="zoomOut">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="zoomIn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="zoomActualSize">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Actual Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zoomFit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Fit Window</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="preview">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="camera_infos">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectLinGuider">
            <property name="text">
             <string>Connect Guider</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="dither">
            <property name="text">
             <string>Dither</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="ZoomableImage" name="imageContainer" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextBrowser" name="logWindow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>782</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="actionSet_Camera"/>
    <addaction name="actionSet_Preview_Interval"/>
   </widget>
   <widget class="QMenu" name="menuGuider">
    <property name="title">
     <string>Guider</string>
    </property>
   </widget>
   <addaction name="menuCamera"/>
   <addaction name="menuGuider"/>
  </widget>
  <action name="actionConnect_LinGuider">
   <property name="text">
    <string>Connect LinGuider</string>
   </property>
  </action>
  <action name="actionDisconnect_LinGuider">
   <property name="text">
    <string>Disconnect LinGuider</string>
   </property>
  </action>
  <action name="actionDither">
   <property name="text">
    <string>Dither</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>Scan</string>
   </property>
  </action>
  <action name="actionSet_Camera">
   <property name="text">
    <string>Set Camera</string>
   </property>
  </action>
  <action name="actionSet_Preview_Interval">
   <property name="text">
    <string>Set Preview Interval...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZoomableImage</class>
   <extends>QWidget</extends>
   <header>zoomableimage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
